apps/web/utils/outlook/filter.ts

```diff
@@ -153,7 +153,7 @@ export async function createCategoryFilter({
       .get();
 
     let category = categories.value.find(
-      (cat: any) => cat.displayName === categoryName,
+      (cat: { displayName: string }) => cat.displayName === categoryName,
```

Comment:
Categories isn't typed automatically so we don't need to type it here?
Or TS thinks it's any type?